src/nft/components/details/detailsV2/LandingPage.test.tsx

```diff
@@ -3,10 +3,26 @@ import { render } from 'test-utils/render'
 
 import { LandingPage } from './LandingPage'
 
+beforeEach(() => {
+  // IntersectionObserver isn't available in test environment
+  const mockIntersectionObserver = jest.fn()
+  mockIntersectionObserver.mockReturnValue({
+    observe: () => null,
+    unobserve: () => null,
+    disconnect: () => null,
+  })
+  window.IntersectionObserver = mockIntersectionObserver
+})
```

Comment:
Consider setting this up as a polyfill. Are you at all worried that some browsers will not support this?

See https://www.npmjs.com/package/intersection-observer.

Reply:
No this is strictly for tests. The IntersectionObserver isn't available in the jest test env. See here: https://stackoverflow.com/questions/44249985/testing-code-that-uses-an-intersectionobserver

Can I word this comment to be more clear?

Reply:
Thanks for the callout. IO is [very well supported](https://caniuse.com/intersectionobserver) by modern browsers so I'm not worried in this case. But if we do happen to find a large portion of our user base is using IE we can look to this solution.